src/include/php_aerospike_types.h

```diff
@@ -131,6 +131,7 @@ enum Aerospike_opt_keys {
 	OPT_POLICY_COMMIT_LEVEL, /* set to one of Aerospike::POLICY_COMMIT_LEVEL_*                                */
 	OPT_TTL,                 /* set to time-to-live of the record in seconds                                  */
 	USE_BATCH_DIRECT,        /* use new batch index protocol if server supports it                            */
+    USE_SERVICES_ALTERNATE,  /* boolean value, default: false                                                 */
```

Comment:
Spacing looks off here.

Reply:
yeah that's a bit weird it looked good in vim and Gitpod but in git it showed it like that :sweat_smile: i will do it according to git diff output and it will be good.